Sources/YUL/String+Indentation.swift

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
Was this moved? I distinctly remember this extension existing somewhere else before

Reply:
It seems a bit extreme to do this but I guess if it's the only solution (we can thank SPM for this because Xcode allows for two modules to include the same source file!). The other, hacky, solution would be to have a file that is shared and symbolic links that link to the file. Then perhaps SPM doesn't consider them as overlapping sources?